test/endToEnd/enums/protoFile/enums.proto

```diff
@@ -10,4 +10,10 @@ message enumTest {
   color a = 1; 
   int64 b = 2;
   repeated color c = 3;
+  fruit d = 4;
```

Comment:
We should test repeated instances of nested types as well

Reply:
Good point! I will add tests for that :)